layouts/partials/compliance/controls.html

```diff
@@ -0,0 +1,29 @@
+<section class="mt-8 py-4 px-4 md:px-0">
+    <div class="container mx-auto max-w-4xl">
+        <h3>What is {{ .Params.service }}?</h3>
+        <p>
+            {{ .Params.whatis }}
+        </p>
+    </div>
+</section>
+<section class="mt-4 py-4 px-4 md:px-0">
+    <div class="container mx-auto max-w-4xl">
+        <h3>What controls can I put in place to evaluate {{ .Params.service }} resources?</h3>
+        <div class="py-4">
+            {{ $key := .Params.service }}
+            {{ $controls := index .Site.Data.compliance "controls" }}
+            {{ $array := index $controls $key }}
+            <ul>
+                {{ range $array }}
+                <li class="text-lg">{{ . }}</li>
+                {{ end }}
+            </ul>
+        </div>
+        <div class="my-5 text-center">
+            <p>Speak to a Solutions Architect to implement policy as code to manage {{ .Params.service }} resources for {{ .Params.framework }} compliance.</p>
```

Comment:
So the way this is laid out you have the CTA and it literally just pops down a few lines to the form. 

Maybe the right way to do this is just have each of the CTAs link off to https://www.pulumi.com/contact/ with the Expert Services option selected. This way it solves the rendering issue.

Reply:
I actually think this is a great idea! I'll see if it is possible though, since this form is injected at load time by hubspot and we may not have access to manipulate the individual elements in that subsection of the dom that they render into, which we will need in order to preselect that option.